### ISSUE TITLE
A* disable crossing via diagonal corners

### ISSUE THREAD

**Issue by playingoDEERUX** [Sahnvour/PathFinder#7]
Is it possible? Title says all.

**Comment by Sahnvour** [comment]
Yes, see for example https://github.com/Sahnvour/PathFinder/blob/master/examples/image.cpp#L138 .
In these loops, for each pixel we look at its (up to) 9 neighbours and add them as children, ie. accessible nodes.
If you don't want to allow moving in diagonal, just add the up/down and left/right neighbours.
